gke-tpu-examples/training/mnist-single-tpu/src/gke-config/standard-tf-mnist-batch-predict-v4.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: mnist-batch-prediction-job
+spec:
+  template:
+    metadata:
+      name: mnist
+      annotations:
+        gke-gcsfuse/volumes: "true"
+    spec:
+      nodeSelector:
+        cloud.google.com/gke-tpu-topology: 2x2x1  # target topology
+        cloud.google.com/gke-tpu-accelerator: tpu-v4-podslice
+      containers:
+      - name: tensorflow
+        image: tensorflow/tensorflow:2.14.0
+        securityContext:
+          privileged: true
+        command: ["/bin/bash", "-c", "--"]
+        args: ["cd /data/tensorflow-mnist-example; pip install -r requirements.txt; python tensorflow_mnist_batch_predict.py"]
```

Comment:
where do we create the container that includes the `tensorflow_mnist_batch_predict.py` script?

Reply:
It should be on the TPU node based on the node selector.

Reply:
oh, the script is accessed from gcs; ok, this is a slightly strange setup, usually we put it inside the container